src/cmd-build

```diff
@@ -8,18 +8,34 @@ dn=$(dirname "$0")
 print_help() {
     cat 1>&2 <<'EOF'
 Usage: coreos-assembler build --help
-       coreos-assembler build [--delay-meta-merge] [--force] [--force-image] [--skip-prune] [--strict] [--tag TAG] [--version VERSION] [TARGET...]
+       coreos-assembler build [OPTIONS]... [TARGET]...
 
   Build OSTree and image base artifacts from previously fetched packages.
   Accepted TARGET arguments:
 
-  - ostree
-  - qemu
-  - metal
-  - metal4k
+  - ostree     Compose an ostree commit
+  - qemu       Also create a QCOW2 image to run with QEMU
+  - metal      Also create a raw disk image
+  - metal4k    Also create a raw disk image for 4K native disks
 
   The "qemu" and "metal" targets imply "ostree". If unspecified, defaults to
   "qemu". They are equivalent to manually running buildextend-[TARGET] after.
+
+  The following options are supported:
+
+  --delay-meta-merge  Set 'coreos-assembler.delayed-meta-merge' in build metadata (Default: false)
+  --force             Always create a new OSTree commit, even if nothing appears to have changed
+  --force-image       Force a build even if there were no ostree changes
```

Comment:
Minor: this is more

```suggestion
  --force-image       Force an image rebuild even if there were no changes to image input
```

(I.e. this just invalidates the image change detection, so it could reuse the same OSTree content if OSTree inputs didn't change.)